src/Power.cpp

```diff
@@ -295,6 +295,22 @@ class AnalogBatteryLevel : public HasBatteryLevel
             raw = raw / BATTERY_SENSE_SAMPLES;
             scaled = operativeAdcMultiplier * ((1000 * AREF_VOLTAGE) / pow(2, BATTERY_SENSE_RESOLUTION_BITS)) * raw;
 #endif
+#if defined(RAK11310)
+            float voltage_raw = operativeAdcMultiplier * ((1000 * AREF_VOLTAGE) / 1023) * raw;
+            
+            LOG_INFO("[POWER DEBUG] AREF=%.2fV, ADC_RAW=%u, ADC_MULT=%.2f", 
+                     (double)AREF_VOLTAGE, 
+                     raw, 
+                     (double)operativeAdcMultiplier);
+            
+            scaled = (voltage_raw * 1.22f) - 0.1f;
```

Comment:
1.22f seems to be a correction factor. can we make this a generic correction macro and remove the #ifdef bracket? In case we need it on more than the 11310